pkg/config/manifest.go

```diff
@@ -89,6 +78,8 @@ type CredentialDefinition struct {
 	Location `yaml:",inline"`
 }
 
+// TODO: use cnab-go's bundle.Location instead, once yaml tags have been added
```

Comment:
Have we been making cnab-go add yaml tags so that we can straight up reuse their structs? Just checked and I guess we have!

I'm surprised that no one complained at us. 😀

Reply:
Yes.  I wanted to prioritize using the canonical structs defined in cnab-go as much as possible/applicable...